inc/RestCommand.php

```diff
@@ -428,7 +478,7 @@ private function do_request( $method, $route, $assoc_args ) {
 			$body = json_decode( $response->body, true );
 			if ( $response->status_code >= 400 ) {
 				if ( ! empty( $body['message'] ) ) {
-					WP_CLI::error( $body['message'] . ' ' . json_encode( array( 'status' => $response->status_code ) ) );
```

Comment:
I didn't. It is still there. There was only a change to no longer put the status at the end so that it matched the output of your test cases.
If you want to change the spec on output, then the code and the test cases need to change. I would recommend that a consistent manner of reporting http status be chosen. In the old codebase, it literally shows the JSON. In your PR feedback above, you want it be not show JSON and be formatted. That would cause inconsistencies in http error code output which I would discourage.